packages/rdma-core/Cargo.toml

```diff
@@ -12,8 +12,8 @@ path = "../packages.rs"
 releases-url = "https://github.com/linux-rdma/rdma-core/releases"
 
 [[package.metadata.build-package.external-files]]
-url = "https://github.com/linux-rdma/rdma-core/releases/download/v54.0/rdma-core-54.0.tar.gz"
-sha512 = "efb98dec017e1eb71ed6f2b652d557d0444c672ff388927bdd724c81bb4baeb5617c81fff609f794c1ff128ab93ae26ed4502bd0ebf14e157737b1b08d0fb4b9"
+url = "https://github.com/linux-rdma/rdma-core/releases/download/v55.0/rdma-core-55.0.tar.gz"
```

Comment:
The timing of the release is such that it doesn't seem like it includes [this EFA fix](https://github.com/linux-rdma/rdma-core/commit/3046ca04a0e2fdacd305cb434e46b75aefbc79a2).

54.1 might be a better choice.

Reply:
Sure, let me update that and re-test.

Reply:
They just released 56.0 which contains the EFA fix. Going to consume that instead.